jekyll/_cci2/code-coverage.md

```diff
@@ -253,9 +411,52 @@ also converted to an `html` page, like many other coverage tools. The Pom file
 above writes to the `target` directory, which you can then store as an artifact
 in your CircleCI `config.yml` file.
 
-Here is a  minimal CI configuration to correspond with the above example:
+Here is a minimal CI configuration to correspond with the above example:
+
+{:.tab.java_example.Cloud}
+```yaml
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+      - image: cimg/openjdk:17.0-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run : mvn test
+      - store_artifacts:
+          path:  target
+```
+
+{:.tab.java_example.Server_3}
+```yaml
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+      - image: cimg/openjdk:17.0-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run : mvn test
```

Comment:
If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`